include_core/omrmemcategories.h

```diff
@@ -85,6 +85,9 @@ typedef struct OMRMemCategorySet {
 #define OMRMEM_CATEGORY_CUDA 0x80000010
 #endif /* OMR_OPT_CUDA */
 
+#define OMRMEM_CATEGORY_CLASSES 2
```

Comment:
I guess they can be put inside a `#if defined()` ?

I know OpenJ9 is using value 2 and 3 for `J9MEM_CATEGORY_CLASSES` and `J9MEM_CATEGORY_CLASSES_SHC_CACHE `, but I see all other memory categories here begins with `0x8000....`. The new defines here do not seem to follow this convention. 

Comment in line 51 says `Add the corresponding entries in omrrasinit.c`. I don't see that change here.

Reply:
Hi @hangshao0 ,

I have incorporated all of the changes you suggested in the latest commit. Meanwhile, introduction of a new OMR_SHARED_CACHE flag required changes to some build related files, which were missed in the previous commit and are now included in the new commit. Could you please review the new commit? 

Thanks :)

Reply:
Thanks for updating the commit. I don't have more comments.